samples/graph-meeting-notification/csharp/MeetingNotification/Model/Notification.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+namespace MeetingNotification.Model
+{
+    using Newtonsoft.Json;
+    using System;
+
+    public class Notifications
+    {
+        [JsonProperty(PropertyName = "value")]
+        public Notification[] Items { get; set; }
+    }
+    public class Notification
```

Comment:
Two classes with almost similar names.
Rename those properly

Reply:
Changed to NotificationContent.